assets/src/edit-story/components/inspector/inspectorProvider.js

```diff
@@ -110,6 +102,12 @@ function InspectorProvider({ children }) {
     }
   }, [currentPage]);
 
+  const ChecklistTabIcon = useCallback(() => {
+    return (
+      <ChecklistIcon checkpoint={currentCheckpoint} height={28} width={28} />
+    );
+  }, [currentCheckpoint]);
```

Comment:
Nit: we could make this an implicit return if we want but lol this comment is probably not necessary at all.

Do we want a callback here or a `useMemo`?
```suggestion
  const ChecklistTabIcon = useMemo(() => (
      <ChecklistIcon checkpoint={currentCheckpoint} height={28} width={28} />
  ), [currentCheckpoint]);
```

Reply:
The `Tab` data as seen on line 158 rejects the memoized version and wants a callback instead. 🤔

Reply:
haha okay, works for me